### PR TITLE
pr_19

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWASt6RZg9QtoM543Hc_###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_19@9
A "NullPointerException" could be thrown; "message" is nullable here.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWASt6RZg9QtoM543Hc9###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_19@6
Complete the task associated to this TODO comment.

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWASt6RZg9QtoM543Hc-###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_19@9
The return value of "length" must be used.

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 2 issues

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

**Review by sns-seb2 — DISMISSED** [review]

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

Comment (sns-seb) [inline src/main/java/org/sonarqube/A.java]: simulating user replying to bot's comments

Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: simulating a non standard comment posted by bot

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: and a 2nd one, just because

**Review by sns-seb2 — APPROVED** [review]

**Pull request closed without merging**